bodhi/server/scripts/bshell.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2018 Red Hat, Inc. and Sebastian Wojciechowski
+#
+# This file is part of Bodhi.
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+"""This script will run a handy python shell initialized with Bodhi models."""
+
+from subprocess import call
+
+import click
+
+
+@click.command()
+@click.version_option(message='%(version)s')
+def get_bodhi_shell():
+    """Run python shell initialized with Bodhi models."""
+    call(['pshell', '/home/vagrant/bodhi/development.ini'])
```

Comment:
We would like this to work on production, so we'll need it to dynamically find the config file. See ```bodhi.server.config.get_configfile()``` as a nice utility to help with this.